### ISSUE TITLE
acceptance test of upgrade

### ISSUE THREAD

**Issue by turadg** [Agoric/Agoric#8961]
## What is the Problem Being Solved?

Our functional tests are all of HEAD. In `/a3p-integration` we now have robust testing of *upgrades* but they only cover the expected changes. We need coverage that the whole product still works as expected after the pending upgrade goes on chain.

## Description of the Design

Have a virtual (empty) proposal that runs after all the others to test functionality that should always work regardless of upgrades performed.

Call it `z:acceptance` so it will always run last.

Its tests should verify that these important functions still work: 
- TBD

```[tasklist]
### Tasks
- [ ] https://github.com/Agoric/agoric-sdk/issues/10049
```

## Test Plan

To be determined with Product

**Comment by turadg** [comment]
Let's be sure to move tests like `core-eval.test.js` from upgrade-next. E.g. to `z:acceptance`.
https://github.com/Agoric/agoric-sdk/pull/9204/#discussion_r1566303891

**Comment by dckc** [comment]
perhaps a relevant subtask:

 - https://github.com/Agoric/agoric-sdk/issues/8884

**Comment by dckc** [comment]
another relevant subtask:

 - https://github.com/Agoric/agoric-sdk/issues/8788

I'm pretty sure such a test was built for upgrade-14, but it no longer runs for new releases.

**Comment by mhofman** [comment]
Yeah I think that test got removed in #9204. Might be worth to re-add it in a better place now that we have #9257

**Comment by mhofman** [comment]
One thing to consider is whether some tests should be able to setup some state before the upgrade so that it can be used by the acceptance layer after the upgrade. That would likely require setting up a "empty core eval" layer before the `upgrade-next` one.

**Comment by LuqiPan** [comment]
This is will be worked on by BytePitch folks(@anilhelvaci and @Jorge-Lopes)

**Comment by dckc** [comment]
stuff from the mainnet1 qualification effort:

https://github.com/Agoric/agoric-sdk/tree/mainnet1B-rc3/packages/deployment/upgrade-test/upgrade-test-scripts
https://github.com/Agoric/agoric-sdk/blob/mainnet1B-rc3/packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/actions.sh
